docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md

```diff
@@ -0,0 +1,226 @@
+
+
+## Adding our repository and auto-deployment
+
+1. Click on your active servers then under the active site, choose the domain name you added. If you are just testing out, then click on the `Default`.
+
+[![Linode API](/docs/assets/configuration-management/adding_repository.gif)](/docs/assets/configuration-management/adding_repository.gif)
+
+2. Click on Apps on the left hand side and then select your `Git Repository`.
+
+[![Linode API](/docs/assets/configuration-management/adding_git_repository.png)](/docs/assets/configuration-management/adding_git_repository.png)
+
+3. Fill in the repository field with your username and repository then select the branch you want to use. Now, click on INSTALL REPOSITORY. Take a closer look how the `Repository` Field is inserted which is  gitlab_username/repository_name_without_.git`. 
+
+eg: 
+
+```
+    dongido/transaction-manager
+
+```
+
+[![Linode API](/docs/assets/configuration-management/git_repository_name.png)](/docs/assets/configuration-management/git_repository_name.png)
+
+
+When you click **INSTALL REPOSITORY** your project in Gitlab will be copied to the Linode server and composer install and other important commands will be done for you by forge under the background. 
+
+If you visit the website again, you will see that the repository is now copied to our Linode server and everything is set up.
+
+[![Linode API](/docs/assets/configuration-management/site_up_and_running.png)](/docs/assets/configuration-management/site_up_and_running.png)
+
+
+## Auto deploy.
+In this era of Continuous integration where we always want to keep making changes or adding features to our live site and want the changes to get deployed immediately.
+
+The next page that you will get after clicking on the `INSTALL REPOSITORY`. There you can set the auto deploy immediately.
+
+
+[![Linode API](/docs/assets/configuration-management/Laravel_forge_deploy.png)](/docs/assets/configuration-management/Laravel_forge_deploy.png)
+
+
+Here, you just need to change the quick deploy to `ON`. Then try making change to your project on gitlab and revisit you site and see your changes appear.
+
+if in any way, you are not on the page, you can get to the deploy page by:
+
+1. Choose thue site that you are working on.
+
+2. Make sure the `quick deploy` option is set to ON.
+
+[![Linode API](/docs/assets/configuration-management/laravel_forge_sites.png)](/docs/assets/configuration-management/laravel_forge_sites.png)
+
+
+## Adding SSL to your domain name
+SSL (Secure Sockets Layer) is the standard security technology for establishing an encrypted link between a web server and a browser. To add SSL:
+
+1. Choose your server on Forge.
+
+2. Select the domain name that you want to add SSL to. 
+
+3. Click on `SSL` from the left options.
+
+
+[![Linode API](/docs/assets/configuration-management/laravel_forge_adding_ssl.png)](/docs/assets/configuration-management/laravel_forge_adding_ssl.png)
```

Comment:
Convention for domain name per style guide is `example.com` and `www.example.com`